lib/common/diagramvisitor.js

```diff
@@ -229,6 +232,16 @@ class DiagramVisitor {
         parameters.fileWriter.writeLine(1, '+ ' + enumValueDeclaration.getName());
     }
 
+    /**
+     * Visitor design pattern
+     * @param {Decorator} decorator - the object being visited
+     * @param {Object} parameters  - the parameter
+     * @protected
+     */
+    visitDecorator(decorator, parameters) {
+        parameters.fileWriter.writeLine(1, '+ ' + decorator.getName());
```

Comment:
What impact does this have on the UML diagram decorators?

Reply:
I can change this to `<<...>>` which is more representative of a decorator

Reply:
Opting to drop decorator representation in the UML